openghg/retrieve/_access.py

```diff
@@ -23,29 +21,33 @@
 
 DataTypes = Union[BoundaryConditionsData, FluxData, FootprintData, ObsColumnData, ObsData]
 multDataTypes = Union[
-    List[BoundaryConditionsData], List[FluxData], List[FootprintData], List[ObsColumnData], List[ObsData]
+    list[BoundaryConditionsData], list[FluxData], list[FootprintData], list[ObsColumnData], list[ObsData]
 ]
 
 
 def _get_generic(
-    sort: bool = True,
+    combine_multiple_inlets: bool = False,
     elevate_inlets: bool = False,
     ambig_check_params: Optional[list] = None,
     **kwargs: Any,
-) -> Any:
+) -> _BaseData:
     """Perform a search and create a dataclass object with the results if any are found.
 
     Args:
         data_class: Type of dataobject to create
-        sort: Sort Dataset during recombination
         elevate_inlets: Elevate the inlet attribute to be a variable within the Dataset
         ambig_check_params: Parameters to check and print if result is ambiguous.
         kwargs: Additional search terms
     Returns:
         dataclass
     """
     from openghg.retrieve import search
-    from openghg.store.spec import define_data_type_classes
+
+    if elevate_inlets is True:
```

Comment:
If `elevate_inlets` gets removed and replaced with` combine_multiple_inlets` this warning can probably be removed.